### PR TITLE
feat: add "Edabit"

### PR DESCRIPTION
**Pull request by Vijaykv5**
#1302 

## Changes proposed

Added "Edabit" to the CodeEditors in website

## Screenshots

Screenshot 2023-09-02 at 12 32 49 AM

### CODE REVIEW

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -152,5 +152,12 @@
     "url": "https://www.jdoodle.com/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+    "name": "Edabit",
+    "description": "Edabit is a Platform to Learn Programming with 10,000+ challenges through Interactive Console Editor",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:158]:
```suggestion
    "description": "Edabit is a platform to Learn Programming with 10,000+ challenges through Interactive Console Editor",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Vijaykv5! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by Anmol-Baranwal** [comment]
@Vijaykv5 
How is this a online code editor?

**Comment by Vijaykv5** [comment]
> @Vijaykv5 How is this a online code editor?

Its a platform for learning with programming languages. Also provides a code editor for the same

**Comment by Anmol-Baranwal** [comment]
> > @Vijaykv5 How is this a online code editor?
> 
> Its a platform for learning with programming languages. Also provides a code editor for the same

What do you think? @CBID2

**Comment by CBID2** [comment]
> > > @Vijaykv5 How is this a online code editor?
> 
> > 
> 
> > Its a platform for learning with programming languages. Also provides a code editor for the same
> 
> 
> 
> What do you think? @CBID2 

Hmmm good question @Anmol-Baranwal. Since it has a IDE and it seems to be where people put their focus the most, I think it's worth adding to that category

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

> As per the suggestion of @CBID2

**Comment by CBID2** [comment]
> 🚀 Looks good to me.
> 
> > As per the suggestion of @CBID2

@Anmol-Baranwal, I actually changed my mind. You brought up a good point on Edabit being more of a learning platform. I think it should go in the...Other Resources category.

File: database/frontend/online-code-editors.json [hunk 2]
```diff
@@ -152,5 +152,12 @@
     "url": "https://www.jdoodle.com/",
     "category": "frontend",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:153]:
```suggestion
    "category": "other",
```

File: database/frontend/online-code-editors.json [hunk 3]
```diff
@@ -152,5 +152,12 @@
...
     "subcategory": "online-code-editors"
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:154]:
```suggestion
    "subcategory": "other-resources"
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Vijaykv5. I changed the categories a bit since the resource seems more like a learning platform

File: database/frontend/online-code-editors.json (hunk 2 above)
Comment (CBID2) [reply database/frontend/online-code-editors.json:153]: You forgot this one @Vijaykv5

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by CBID2** [comment]
@Anmol-Baranwal, this is ready for another review.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Pull request merged**